config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  get '/cart', to: 'cart#show'
-  post '/cart', to: 'cart#create'
+  get '/charges', to: 'charges#show'
+  post '/charges', to: 'charges#create'
```

Comment:
Not sure why this is around. Out of scope.

Reply:
Yes it is, we are taking it out from this PR and making a separate PR for it 👍